package.json

```diff
@@ -10,13 +10,13 @@
     "@jazzer.js/core": "^2.1.0",
     "@metalsmith/layouts": "^2.7.0",
     "@metalsmith/markdown": "^1.10.0",
-    "@napi-rs/canvas": "^0.1.60",
+    "@napi-rs/canvas": "^0.1.62",
```

Comment:
Please update the following line as well, such that we keep these entries consistent:
https://github.com/mozilla/pdf.js/blob/8a8b69f456be90010c4892d24fddf31d9f97a68d/gulpfile.mjs#L2256